### PR TITLE
Implementing In Defense of Kamino

### PR DESCRIPTION
**Pull request by supercodepoet**
image

### CODE REVIEW

File: test/server/cards/03_TWI/events/InDefenseOfKamino.spec.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+describe('In Defense of Kamino', function () {
+    integration(function (contextRef) {
+        describe('In Defense of Kamino\'s ability', function () {
+            it('should give each friendly Republic unit Restore 2 and: "When Defeated: Create a Clone Trooper token."', function () {
```
Comment (AMMayberry1) [inline test/server/cards/03_TWI/events/InDefenseOfKamino.spec.ts:4]: nit: can merge the `describe` into the `it`

File: test/server/cards/03_TWI/events/InDefenseOfKamino.spec.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                        spaceArena: ['gladiator-star-destroyer']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.inDefenseOfKamino);
+                expect(context.player2).toBeActivePlayer();
+
+                context.player2.clickCard(context.gladiatorStarDestroyer);
```
Comment (AMMayberry1) [inline test/server/cards/03_TWI/events/InDefenseOfKamino.spec.ts:23]: Can start the base with damage on it like this:

```ts
base: { card: 'echo-base', damage: 5 }
```

File: test/server/cards/03_TWI/events/InDefenseOfKamino.spec.ts (hunk 1 above)
Comment (supercodepoet) [reply test/server/cards/03_TWI/events/InDefenseOfKamino.spec.ts]: done

File: test/server/cards/03_TWI/events/InDefenseOfKamino.spec.ts (hunk 2 above)
Comment (supercodepoet) [reply test/server/cards/03_TWI/events/InDefenseOfKamino.spec.ts]: gotcha, done

Comment (supercodepoet) [reply test/server/cards/03_TWI/events/InDefenseOfKamino.spec.ts]: Is it better to do something like that and use passAction() or simulate actual gameplay?

**Review by AMMayberry1 — APPROVED** [review]

**Pull request merged**